### DISCUSSION TITLE
Chia keys generate coinbase address

### DISCUSSION THREAD

**Discussion by 2hq53hs8tur** [Chia-Network/chia-blockchain#4233] [Support]
Hi! Excited about the Chia project. I'm using chia from command line, and when I call "chia keys generate" I get "Setting the xch destination address for coinbase reward to...". Should I somehow set this address to my personal coinbase address, or leave it as is? Thanks

**Comment by 2hq53hs8tur** [comment]
Also, as a follow-up, do I need to call `chia start node` or `chia start farmer` before I begin plotting? What is the ordering of `chia plots create`, `chia start node`, and `chia start farmer`? Thanks!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
